BookingsApi/BookingsApi.IntegrationTests/Api/V2/Hearings/UpdateHearingsInGroupV2Tests.cs

```diff
@@ -108,8 +108,11 @@ public async Task should_update_hearings_in_group()
                 AssertParticipantsUpdated(updatedHearing, requestHearing);
                 AssertEndpointsUpdated(updatedHearing, requestHearing);
                 AssertJudiciaryParticipantsUpdated(updatedHearing, requestHearing);
-                AssertEventsPublished(updatedHearing, requestHearing, existingParticipantsUpdated: 1);
+                AssertEventsPublished(updatedHearing, requestHearing);
             }
+            var requestFirstHearing = request.Hearings.First(x => x.HearingId == updatedHearings[0].Id);
+            AssertParticipantEvents(updatedHearings[0], requestFirstHearing, existingParticipantsUpdated: 1);
```

Comment:
I think we should be asserting on the multi day notifications events here, rather than the participant events.

also, we should make the same changes to the V1 tests